core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -54,7 +54,12 @@ class TaskInfo(
    * accumulable to be updated multiple times in a single task or for two accumulables with the
    * same name but different IDs to exist in a task.
    */
-  val accumulables = ListBuffer[AccumulableInfo]()
+  lazy val accumulables: ListBuffer[AccumulableInfo] = ListBuffer(_accumulables: _*)
```

Comment:
Actually, though, if we're going to expose a public API for this at all then I think it should be read-only. Let me go ahead and refactor this to expose a read-only `Seq`.